drivers/misc/ethos_u/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Arm Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(
+    init.c)
```

Comment:
please fix formatting here, a single line is enough

Reply:
Fixed.